packages/sanity/src/core/store/bundles/__workshop__/ReleaseForm.tsx

```diff
@@ -0,0 +1,135 @@
+import {COLOR_HUES} from '@sanity/color'
```

Comment:
This is just a copy from the prototype, that's why I'm adding it in `__workshop__`

Reply:
To add a note for future proof: we spoke about this on slack, this has changed slightly ([naming and structure](https://github.com/sanity-io/sanity/blob/0162c683cd43ee01c443d3e28e84ab88b7c2e876/packages/sanity/src/core/versions/components/dialog/BundleForm.tsx)) but we can update this once both our PRs are merged

Reply:
Thanks!